docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+
+        ```console
+        semgrep ci
+        ```
+
+    </TabItem>
+
+    <TabItem value='Docker'>
+
+    :::info Prerequisites
+    Ensure that you have the [Docker installed](https://docs.docker.com/desktop/) before proceeding.
+    :::
+
+    1. Pull the latest image and confirm the version:
+
+        ```console
+        docker pull returntocorp/semgrep
+
+        # confirm version
+        docker run --rm returntocorp/semgrep semgrep --version
+        ```
+
+    2. For users running Docker on **macOS or Linux** Docker:
+
+        1. Log in to your Semgrep account (running this command will launch a browser window, but you can also use the link that's returned in the CLI to proceed):
+
+            ```console
+            docker run -it returntocorp/semgrep semgrep login
+            ```
+
+        2. In the **Semgrep CLI login**, click **Activate** to proceed. Return to the CLI and copy the login token that's shown.
+
+        3. Navigate into the root of your repo, and run your first scan (be sure to substitute <code><span class="placeholder">YOUR_TOKEN</span></code> with the login token value you copied in the previous step): 
+
+            ```console
+            docker run -e SEMGREP_APP_TOKEN=YOUR_TOKEN --rm -v "${PWD}:/src" returntocorp/semgrep semgrep ci
+            ```
+
+            The provided `-v` option mounts the current directory into the container to be scanned. Navigate into a different repo or provide a specific local directory in the command to scan a different project.
+    
+    3. For users running Docker on **Windows**:
+
+        1. Log in to your Semgrep account (running this command will launch a browser window, but you can also use the link that's returned in the CLI to proceed):
+
+            ```console
+            docker run -it returntocorp/semgrep semgrep login
+            ```
+
+        2. In the **Semgrep CLI login**, click **Activate** to proceed. Return to the CLI, and copy the login token that's shown.
+
+        3. Navigate into the root of your repo, and run your first scan (be sure to substitute <code><span class="placeholder">YOUR_TOKEN</span></code> with the login token value you copied in the previous step): 
+
+            ```console
+            docker run -e SEMGREP_APP_TOKEN=YOUR_TOKEN --rm -v "%cd%:/src" returntocorp/semgrep semgrep ci
+            ```
+
+            The provided `-v` option mounts the current directory into the container to be scanned. Navigate into a different repo or provide a specific local directory in the command to scan a different project.
+
+    </TabItem>
+
+    </Tabs>
+
+4. Once you've scanned your first application, return to the Cloud Platform, and click **View findings** to see the security vulnerabilities in your project. Alternatively, you can view your results in the Cloud Platform's **Dashboard** page. For detailed information, click **Code** to access your SAST findings or **Supply Chain** to access your SCA findings.
```

Comment:
Noting this line down pending results of our To Use Full Name or Not To Use Full Name discussion.